scripts/voqutil

```diff
@@ -0,0 +1,337 @@
+    else:
+        chassis_app_db_keys = chassis_app_db.keys(chassis_app_db.CHASSIS_APP_DB, "SYSTEM_LAG_MEMBER_TABLE|%s:%s" % system_lag_name, system_lag_member_name)
+
+    return chassis_app_db_keys
+
+# ========================== VOQ system port status logic ==========================
+
+header_system_port = ['System Port Name', 'Port Id', 'Switch Id', 'Core', 'Core Port', 'Speed']
+
+class SystemPortStatus(object):
+
+    def __init__(self, system_port_name, namespace_option, display_option):
+        """
+        Class constructor method
+        :param self:
+        :param intf_name: string of system port name
+        :return:
+        """
+        self.db = None
+        self.config_db = None
+        self.system_port_name = system_port_name
+        self.table = []
+        self.multi_asic = multi_asic_util.MultiAsic(
+            display_option, namespace_option)
+
+    def display_systemport_status(self):
+        self.get_systemport_status()
+        sorted_table = natsorted(self.table)
+        print(tabulate(sorted_table,
+                       header_system_port,
+                       tablefmt="simple",
+                       stralign='right'))
+
+    def generate_systemport_status(self):
+        """
+            Generate system port status output
+        """
+
+        i = {}
+        table = []
+        key = []
+
+        intf_fs = parse_systemintf_in_filter(self.system_port_name)
+        #
+        # Iterate through all the keys and append system port's associated info to
+        # the result table.
+        #
+        for i in self.appl_db_keys_system_port:
+            key = re.split(':', i, maxsplit=1)[-1].strip()
+            if self.system_port_name is None or key in intf_fs:
+                table.append((key,
+                        appl_db_system_port_status_get(self.db, key, SYSTEM_PORT_ID),
+                        appl_db_system_port_status_get(self.db, key, SYSTEM_PORT_SWITCH_ID),
+                        appl_db_system_port_status_get(self.db, key, SYSTEM_PORT_CORE),
+                        appl_db_system_port_status_get(self.db, key, SYSTEM_PORT_CORE_PORT),
+                        appl_db_system_port_status_get(self.db, key, SYSTEM_PORT_SPEED)))
+
+        return table
+
+    @multi_asic_util.run_on_multi_asic
+    def get_systemport_status(self):
+        self.appl_db_keys_system_port = appl_db_keys_get_system_port(self.db, None)
+        if self.appl_db_keys_system_port:
+            self.table += self.generate_systemport_status()
+
+# ========================== voq system neigh status logic ==========================
+
+header_system_neigh = ['System Port Interface', 'Neighbor', 'MAC', 'Encap Index']
+
+class SystemNeighStatus(object):
+
+    def __init__(self, namespace_option, display_option):
+        self.db = None
+        self.config_db = None
+        self.table = []
+        self.multi_asic = multi_asic_util.MultiAsic(
```

Comment:
Looks `self.multi_asic` like this is not used ?

Reply:
Fixed. Removed since this is not  used